pom.xml

```
@@ -67,7 +67,7 @@
   <properties>
     <revision>2.6.3</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.308</jenkins.version>
+    <jenkins.version>2.303.1</jenkins.version>
```

Still believe this is unnecessary (generally prefer to build & test against the version which actually forced the original change) but :shrug: either should be OK.

---

At this point we can even use the next LTS baseline, but eh.

---

So
```suggestion
    <jenkins.version>2.308</jenkins.version>
```
and just reword PR to say that it is improving icon display, or whatever it is. Not really up to speed on icon handling in Jenkins so I am not sure what the practical impact is of this change. Do not care about publishing updates for 2.303.x users; if they want updates for anything, start by moving to 2.319.x.

---

Thanks for the feedback. Core version reverted and decription updated.